content/documentation/features/index.adoc

```diff
@@ -22,25 +22,25 @@ menu:
 
 == Overview
 
-Kiali helps you define, validate, and observe your Istio service mesh. Kiali provides answers to the questions: What are the microservices in my service mesh, and how are they connected?
+Kiali helps you define, validate, and observe the connections and microservices of your Istio service mesh.
 
-Kiali works with Istio in OpenShift or Kubernetes. It visualizes the service mesh topology and provides visibility into features like request routing, circuit breakers, request rates, latency and more. Kial offers insights about the mesh components at different levels, from abstract Applications to Services and Workloads.
+Kiali works with Istio in OpenShift or Kubernetes. It visualizes the service mesh topology and provides visibility into features like request routing, circuit breakers, request rates, latency and more. Kiali offers insights about the mesh components at different levels, from abstract applications to services and workloads.
```

Comment:
This could go either way, the capitalization matches the Application/Service/Workloads navigation in the user interface.